moment_kinetics/src/time_advance.jl

```diff
@@ -288,11 +288,12 @@ function setup_time_advance!(pdf, vz, vr, vzeta, vpa, vperp, z, r, vz_spectral,
     begin_serial_region()
     vperp_advect = setup_advection(n_ion_species, vperp, vpa, z, r)
     # initialise the vperp advection speed
+    # note that z_advect and r_advect are arguments of update_speed_vperp!
```

Comment:
Not sure this comment adds anything. Suggest either remove it, or add an explanation of why it is important to note this.

Reply:
This means that we should make sure to calculate the z_advect and r_advect speeds before we try to use vperp_advect. Is this extra comment enough?

Reply:
Yes, perfect.